moped-editor/src/views/projects/projectView/ProjectComponents/ProjectComponentsList.js

```diff
@@ -34,10 +36,10 @@ const ProjectComponentsList = ({
     setIsClickedComponentRelated(false);
     // Clear clickedComponent and draftEditComponent when we are not selecting for edit
     if (isExpanded(component)) {
-      setClickedComponent(null);
+      makeClickedComponentUpdates(null);
```

Comment:
First of many places where I bundled updating `clickedComponent` state and updating the URL search param since we would want the param to show anytime a UI element is clicked that brings the clicked component into focus (zoom, list expand, map feature click).